Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs

```diff
@@ -46,7 +46,7 @@ private void SerializeSpan(StreamWriter writer, Span span)
             writer.Write(openingBrace);
             writer.WriteField(id, NumberUtils.EncodeLongToLowerHexString(span.SpanState.SpanId));
             writer.Write(comma);
-            writer.WriteField(name, span.Name ?? SerializerUtils.DefaultRpcMethodName);
+            writer.WriteField(name, span.Name == null ? SerializerUtils.ToEscaped(span.Name) : SerializerUtils.DefaultRpcMethodName);
```

Comment:
I think you meant the opposite. If the name is *not* null then escape it

Reply:
my bad